File: javascript/javascript-check-element-is-in-array/example2.js
// https://codevscolor.com/javascript-check-element-is-in-array

const isInArray = (arr, n) => {
    return arr.indexOf(n) !== -1;
}

console.log(`2 is in [1, 2, 3, 4, 5, 6]: ${isInArray([1, 2, 3, 4, 5, 6], 2)}`);
console.log(`7 is in [1, 2, 3, 4, 5, 6]: ${isInArray([1, 2, 3, 4, 5, 6], 7)}`);

File: swift/swift-odd-even-check/example2.swift
// https://codevscolor.com/swift-odd-even-check

var arr = [1,2,3,4,5,6]

for n in arr{
    if(n.isMultiple(of: 2)){
        print("\(n) is even")
    }else{
        print("\(n) is odd")
    }
}